src/test/java/com/google/dicomwebfuse/dao/FuseDaoHelperTest.java

```diff
@@ -111,20 +99,6 @@ void testShouldReturnFiveThousandStudiesIfStudiesCountFiveThousand()
     assertEquals(expectedStudiesCount, actualStudyList.size());
   }
 
```

Comment:
Please add new test cases for navigating to a study that's not present in the listing.

Reply:
it's done in src/test/java/com/google/dicomwebfuse/fuse/DicomFuseHelperTest.java  https://github.com/GoogleCloudPlatform/healthcare-api-dicom-fuse/pull/28/files/463d95ee32b65c83d4a621575be8e7f72278e855#diff-fb5541c3c9fa2fdbb8cdf87c64d2ce54R1 because changes are made in DicomFuseHelper class

Reply:
UPD: https://github.com/GoogleCloudPlatform/healthcare-api-dicom-fuse/pull/28/files#diff-fb5541c3c9fa2fdbb8cdf87c64d2ce54R1